solidity/contracts/BancorNetwork.sol

```diff
@@ -430,7 +431,7 @@ contract BancorNetwork is IBancorNetwork, TokenHolder, ContractRegistryClient, F
         }
         // EtherToken
         else if (etherTokens[_sourceToken]) {
-            // claim the tokens - if the source token is address(0), this call will fail
+            // claim the tokens - if the source token is ETH_RESERVE, this call will fail

```

Comment:
should be more like `if the source token address is ETH_RESERVE_ADDRESS....`

Reply:
@yudilevi : Please see updated code.